### PR TITLE
Intern: Build Tool Decoupling CLI

### PR DESCRIPTION
**Pull request by akshaymahajans**
### Summary

  I replaced the webpack_json management command with a command line tool, build_tools/webpack_json.py, that allows you to initialize and build specified kolibri plugins, rather than building every single plugin (as the management command did).

image

…

### Reviewer guidance

 * A reviewer can attempt to run the command " python webpack_json.py" with either of the following options:
1. -i option = enter an input file with a list of kolibri plugins you'd like to build.
2. -b option = enter a python list of plugins, or a space-separated list of plugins to build.

…

### References

 -Related to the management command webpack_json.py

…

----

### Contributor Checklist

- [- ] Contributor has fully tested the PR manually
- [- ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,4 +24,4 @@ COPY . /kolibri
 
 VOLUME /kolibridist/  # for mounting the whl files into other docker containers
 # add buildkite pipeline specific installation here:
-CMD cd /kolibri && pip install -r requirements/dev.txt && pip install -r requirements/build.txt && yarn install && make dist pex && cp /kolibri/dist/* /kolibridist/
+CMD cd /kolibri && pip install -r requirements/dev.txt && pip install -r requirements/build.txt && pip install -e . && yarn install && make dist pex && cp /kolibri/dist/* /kolibridist/
```
Comment (rtibbles) [inline Dockerfile:27]: @aronasorman @lyw07 do you have any concerns about doing this here? It is required in order to access some internal Kolibri modules in the build tools.

Comment (lyw07) [reply Dockerfile:27]: I'm not really sure about this... But I get an error running the pex file. It's
```
WebpackError at /setup_wizard/
Webpack build file missing, front-end assets cannot be loaded
```

Comment (rtibbles) [reply Dockerfile:27]: Looks like it is not building any plugins, only the core bundle!

**Comment by rtibbles** [comment]
Superseded by #3542

**Pull request closed without merging**